bom/pom.xml

```diff
@@ -41,7 +41,7 @@ THE SOFTWARE.
   <properties>
     <guavaVersion>11.0.1</guavaVersion>
     <slf4jVersion>1.7.30</slf4jVersion>
-    <stapler.version>1.262</stapler.version>
+    <stapler.version>1.263-rc1502.c0b75c89ee9d</stapler.version><!-- TODO https://github.com/stapler/stapler/pull/211 -->
```

Comment:
Do we need this released before merging?

Reply:
Yes.